data/brands/amenity/fuel.json

```diff
@@ -33,6 +33,36 @@
         "name": "1-2-3"
       }
     },
+    {
+      "displayName": "ميد",
+      "locationSet": {"include": ["sa"]},
+      "tags": {
+        "amenity": "fuel",
+        "brand": "ميد",
+        "brand:ar": "شركة ميد التجارية",
+        "brand:en": "MEED Trading Company",
+        "brand:wikidata": "Q105492269",
+        "brand:wikipedia": "ar:شركة ميد",
+        "name": "شركة ميد",
```

Comment:
Normally the `name` tag should match the `name:ar` tag in this case..
I think some validators (KeepRight and JOSM?) may give a warning if there are localized names but none of them matches `name`.